City-of-Helsinki/drupal-infofinland
# UHF-10876 askem

# [UHF-10876](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10876)


Since webform is only used with the custom-react-contact from which is being deleted, we can uninstall webform-module

## What was done

Remove the original contact form.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10876`
  * `make fresh`
* Run `make drush-cr`

## How to test

- See that the site can be setup normally after webform-module was removed
- Admin side should work without problems and status page should load.



[UHF-10876]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10876?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
